src/test/java/org/folio/consortia/controller/UserTenantControllerTest.java

```diff
@@ -207,33 +207,6 @@ void getUserTenant_shouldReturnExpectedStatusAndError(String path, String errorC
         jsonPath("$.errors[0].code", is(errorCode)));
   }
 
-  private ConsortiumEntity createConsortiumEntity() {
-    ConsortiumEntity consortiumEntity = new ConsortiumEntity();
```

Comment:
we have excluded domain package from sonarcloud code coverage, so we don't need it anymore